alf/trainers/policy_trainer.py

```diff
@@ -736,14 +731,17 @@ def play(root_dir,
                                                max_episode_length > 0)
 
         if recorder:
+            # Note that because we capture the frame *after* calling ``env.step()``
+            # so the ``policy_step`` here contains prediction info of the frame
+            # *before* the currently rendered frame by the side. (i.e., pred info is
+            # always delayed by one frame).
```

Comment:
Seems to me we should make a change so that there is no delay.

Reply:
changed